src/main/java/edu/unc/lib/boxc/migration/cdm/services/CdmExportFilesService.java

```diff
@@ -76,8 +77,17 @@ public String exportUnmapped() throws IOException {
                         currentUnmapped++;
                         // Figure out name of associated file and download it
                         var filename = retrieveSourceFileName(dbConn, origMapping);
-                        var filePath = imageDir.resolve(filename).toString();
+                        String filePath;
+                        // Pdf and image cpd objects are located in different places
+                        if (retrieveEntryTypeField(dbConn, origMapping) != null
+                                && retrieveEntryTypeField(dbConn, origMapping)
+                                .equals(CdmIndexService.ENTRY_TYPE_DOCUMENT_PDF)) {
+                            filePath = pdfDir.resolve(origMapping.getCdmId() + "/index.pdf").toString();
+                        } else {
+                            filePath = imageDir.resolve(filename).toString();
+                        }
                         var destPath = exportSourceFilesPath.resolve(filename);
```

Comment:
What does the filename look like for these pdf files? Was just wondering if they are cpd extensions or pdf, and if it might be weird if we download the index.pdf file as a cpd file

Reply:
I think the pdf filenames are index.pdf. I'll fix it.